diff --git a/node_modules/@zenstackhq/tanstack-query/runtime-v5/svelte.d.ts b/node_modules/@zenstackhq/tanstack-query/runtime-v5/svelte.d.ts
index 1fd07e7..2c30f43 100644
--- a/node_modules/@zenstackhq/tanstack-query/runtime-v5/svelte.d.ts
+++ b/node_modules/@zenstackhq/tanstack-query/runtime-v5/svelte.d.ts
@@ -29,7 +29,7 @@ declare function getHooksContext(): {
  * @param fetch The fetch function to use for sending the HTTP request
  * @returns useQuery hook
  */
-declare function useModelQuery<TQueryFnData, TData, TError>(model: string, url: string, args?: unknown, options?: StoreOrVal<Omit<CreateQueryOptions<TQueryFnData, TError, TData>, 'queryKey'>> & ExtraQueryOptions, fetch?: FetchFn): _tanstack_svelte_query_v5.CreateQueryResult<unknown, Error>;
+declare function useModelQuery<TQueryFnData, TData, TError>(model: string, url: string, args?: unknown, options?: StoreOrVal<Omit<CreateQueryOptions<TQueryFnData, TError, TData>, 'queryKey'>> & ExtraQueryOptions, fetch?: FetchFn): _tanstack_svelte_query_v5.CreateQueryResult</* unknow */ TData, Error>;
 /**
  * Creates a svelte-query infinite query.
  *
